stock_report_deliveryslip_adj/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Report Delivery Slip Adjustments",
+    "version": "12.0.1.0.0",
+    "category": "Reporting",
+    "license": "AGPL-3",
+    "author": "Quartile",
+    "website": "https://www.quartile.co",
+    "depends": [
+        "sale_stock",
+        "sale_order_line_customer_reference",
+        "stock_picking_report_valued",
+        "stock_secondary_unit",
+        "sale_order_delivery_date",
```

Comment:
nitpick: List modules in following orders:

- Odoo standard modules > others
  - alphabetical order

```suggestion
        "sale_stock",
        "sale_order_delivery_date",
        "sale_order_line_customer_reference",
        "stock_picking_report_valued",
        "stock_secondary_unit",
```